pkg/openscap/openscap_test.go

```diff
@@ -0,0 +1,70 @@
+package openscap
+
+import (
+	"fmt"
+	//docker "github.com/fsouza/go-dockerclient"
+	"testing"
+)
+
+type testScanner struct {
+	defaultScanner
+
+	getRHELDistOriginal bool
```

Comment:
yes, you're right.  Let's disregard for now.  We can revisit it later if necessary.